exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -105,6 +106,13 @@ func (c *CoinbasePro) SetDefaults() {
 				GlobalResultLimit: 300,
 			},
 		},
+		Subscriptions: subscription.List{
+			{Enabled: true, Channel: "heartbeat"},
```

Comment:
That's unacceptable.

It highlights a problem I have with the lack of distinction between default config, user config(s), and runtime reality.
We should not be overloading the runtime b.Features and then back merging it into Config.
I'll come back to this in another PR, since it's obviously a bigger topic.

Reply:
Fixed ecd229c

Reply:
Now its gone the other way. Testing going through each exchange. On first runthrough, no endpoints were subscribed to and my config ended up with the following:
```json
"subscriptions": [
     {
      "enabled": false,
      "channel": "heartbeat"
     },
     {
      "enabled": false,
      "channel": "level2_batch"
     },
     {
      "enabled": false,
      "channel": "ticker"
     },
     {
      "enabled": false,
      "channel": "user",
      "authenticated": true
     },
     {
      "enabled": false,
      "channel": "matches"
     }
    ]
```

Everything worked well when setting it to `true` in my config though. So something is setting it to `false` /nothing is setting it to `true when _initially_ adding data to the config